WordPress/Classes/Extensions/WPStyleGuide+Search.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import WordPressShared
+
+extension WPStyleGuide {
+
+    public class func configureSearchBar(searchBar: UISearchBar) {
+        searchBar.backgroundColor = UIColor.blueColor()
+        searchBar.accessibilityIdentifier = NSLocalizedString("Search", comment: "")
+        searchBar.autocapitalizationType = .None
```

Comment:
Do we want to set `autocorrectionType` to `.No` here? I myself always don't prefer autocorrect while typing something specific, no idea how common that is.